src/test/scala/TiebreakTests.scala

```diff
@@ -0,0 +1,193 @@
+          Round(6).lose(mega, irina),
+        )
+      case "statmatt" =>
+        List(
+          Round(0).lose(matt, derarzt),
+          Round(1).win(matt, fun),
+          Round(2).lose(matt, berserk),
+          Round(3).lose(matt, irina),
+          Round(4).bye(matt),
+          Round(5).lose(matt, rapha),
+          Round(6).win(matt, manka),
+        )
+      case "mankalacz" =>
+        List(
+          Round(0).lose(manka, ayala),
+          Round(1).lose(manka, mega),
+          Round(2).bye(manka),
+          Round(3).win(manka, fun),
+          Round(4).lose(manka, rapha),
+          Round(5).lose(manka, berserk),
+          Round(6).lose(manka, matt),
+        )
+      case "FunWithFlags" =>
+        List(
+          Round(0).noOpponentWin(fun, egor),
+          Round(1).lose(fun, matt),
+          Round(2).lose(fun, irina),
+          Round(3).lose(fun, manka),
+          Round(4).lose(fun, mega),
+          Round(5).bye(fun),
+          Round(6).lose(fun, derarzt),
+        )
+      case "egormmaksymov341" =>
+        List(
+          Round(0).noShowLoss(egor, fun),
+          Round(1).withdrawn(egor),
+          Round(2).withdrawn(egor),
+          Round(3).withdrawn(egor),
+          Round(4).withdrawn(egor),
+          Round(5).withdrawn(egor),
+          Round(6).withdrawn(egor),
+        )
+    }
+  List()
+}
+
+class TiebreakTests extends Specification with ValidatedMatchers {
+  val draughts = new TournamentEY1k6otN();
+  val tourney  = new Tiebreak(draughts);
+  "derarzt results" should {
+    tourney.score(draughts.derarzt) must_== 14
+    tourney.sonnenbornBerger(draughts.derarzt) must_== 52
+    tourney.buccholz(draughts.derarzt) must_== 52
+  }
+  "ayala results" should {
+    tourney.score(draughts.ayala) must_== 12
+    tourney.sonnenbornBerger(draughts.ayala) must_== 34
+    tourney.buccholz(draughts.ayala) must_== 58
+  }
+  "rapha results" should {
+    tourney.score(draughts.rapha) must_== 10
+    tourney.sonnenbornBerger(draughts.rapha) must_== 30
+    tourney.buccholz(draughts.rapha) must_== 56
+  }
+  "irina results" should {
+    tourney.score(draughts.irina) must_== 8
+    tourney.sonnenbornBerger(draughts.irina) must_== 22
+    tourney.buccholz(draughts.irina) must_== 58
+  }
+  "berserker results" should {
+    tourney.score(draughts.berserk) must_== 6
+    tourney.sonnenbornBerger(draughts.berserk) must_== 10
+    tourney.buccholz(draughts.berserk) must_== 59
+  }
+  "megamau results" should {
+    tourney.score(draughts.mega) must_== 6
+    tourney.sonnenbornBerger(draughts.mega) must_== 8
+    tourney.buccholz(draughts.mega) must_== 57
+  }
+  "statmatt results" should {
+    tourney.score(draughts.matt) must_== 6
+    tourney.sonnenbornBerger(draughts.matt) must_== 8
+    tourney.buccholz(draughts.matt) must_== 50
+  }
+  "mankalacz results" should {
+    tourney.score(draughts.manka) must_== 4
+    tourney.sonnenbornBerger(draughts.manka) must_== 4
+    tourney.buccholz(draughts.manka) must_== 48
```

Comment:
I think buccholz should be 44 (missing 5 half points from playing players with byes) and wrong own bye calc

Reply:
own bye is correct, so should be 43